mozetl/taar/taar_lite_guidranking.py

```diff
@@ -0,0 +1,75 @@
+"""
+This ETL job computes the installation rate of all addons and then
+cross references against the whitelist to compute the total install
+rate for all whitelisted addons.
+"""
+
+
+import click
+import json
+import logging
+from pyspark.sql import SparkSession
+from pyspark.sql import SQLContext
+import mozetl.taar.taar_utils as taar_utils
+
+
+OUTPUT_BUCKET = 'telemetry-parquet'
+OUTPUT_PREFIX = 'taar/lite/'
+OUTPUT_BASE_FILENAME = 'guid_install_ranking'
+
+
+def extract_telemetry(spark):
+    """ load some training data from telemetry given a sparkContext
+    """
+    sc = spark.sparkContext
+
+    sqlContext = SQLContext.getOrCreate(sc)
+    frame = sqlContext.sql("""select addon_guid, count(*) as install_count from
+    (select client_id, explode(active_addons[0]) as (addon_guid, addon_row)
+    from longitudinal
+    WHERE normalized_channel='release' AND
+    build IS NOT NULL AND
+    build[0].application_name='Firefox') group by addon_guid
+    """)
+    return frame
+
+
+def transform(frame):
+    json_frame = frame.toJSON()
+    result_list = json_frame.collect()
```

Comment:
How big do you expect this to be? Did you try this job? Does `collect` trigger an OOM situation?